frontend/src/util/validators.js

```diff
@@ -186,7 +185,7 @@ function validateSpecialCharactersForAddress(input, length) {
  * Used to validate that Mailing Address line 1 must be completed if any other Mailing Address(Line 2, Line 3, Line 4, PostalCode) is complete
  */
 export function validateMailingAddress() {
-  return this.mailingAddress.addressLine2 !== '' || this.mailingAddress.addressLine3 !== '' || this.mailingAddress.addressLine4 !== '' || this.mailingAddress.postalCode !== ''
+  return this.mailingAddress.addressLine2 !== '' || this.mailingAddress.city !== '' || this.mailingAddress.province !== '' || this.mailingAddress.postalCode !== ''
```

Comment:
addressline3 and addressline4 still being used by other transactions. Create a new function for now